server/config/diff_test.go

```diff
@@ -131,7 +133,9 @@ func TestDiffSanitized(t *testing.T) {
 					BaseVal: model.Config{},
 					ActualVal: func() model.Config {
 						cfg := defaultConfigGen()
-						cfg.Sanitize()
+						if err := cfg.Sanitize(); err != nil {
+							t.Errorf("Failed to sanitize config: %v", err)
+						}
```

Comment:
nit: we can use `require` assertions.
```suggestion
						err := cfg.Sanitize()
						require.NoError(t, err)
```